ros/src/computing/planning/common/lib/openplanner/op_utility/include/op_utility/UtilityH.h

```diff
@@ -41,7 +41,7 @@ class UtilityH
 	static double GetTimeDiffNow(const struct timespec& old_t);
 	static double GetTimeDiff(const struct timespec& old_t,const struct timespec& curr_t);
 	static std::string GetDateTimeStr();
-	static int tsCompare (struct  timespec  time1,   struct  timespec  time2, int micro_tolerance = 10);
+	static int tsCompare(struct timespec time1, struct timespec time2, int nanoTolerance = 10);
```

Comment:
Please do fixes like this in a separate PR so we can review them separately.

Reply:
Thanks, I have reverted the argument name fix, even though it feels bad leaving 'micro' there while the expected unit is nanoseconds :) but I'll indeed  make a separate PR for that. Are there any other changes you refer to? I don't expect you mean the actual bug fixes to make the tests pass?

Reply:
I think opinions would probably be divided on whether adding tests and fixing the code in response to those tests should be separate PRs. :) In this case, I have enough PRs to review already...